int16 pwm1
int16 pwm2
int16 pwm3
int16 angle1
int16 angle2
int16 fpga
---
int8 result
